cypress/integration/AppAnalytics/BasicSearchResultsPageLoad.feature

```diff
@@ -29,4 +29,4 @@ Feature: Basic search results Page Load
             | page.additionalDetails.numberResults | (int)11189                                     |
             | page.additionalDetails.searchKeyword | tumor                                          |
             | page.additionalDetails.itemsPerPage  | (int)20                                        |
-            | page.additionalDetails.pageNum       | (int)1                                         |
\ No newline at end of file
+            | page.additionalDetails.pageNum       | (int)0                                         |
```

Comment:
one more thing - this should still be a '1', an actual page number not it's index